### PR TITLE
BZ2100397: Add resize & snapshot to vSphere

### PR DESCRIPTION
**Pull request by lpettyjo**
BZ2100397: Add resize & snapshot to vSphere

Version(s):4.11+

Issue:https://bugzilla.redhat.com/show_bug.cgi?id=2100397

Link to docs preview: http://file.rdu.redhat.com/lpettyjo/BZ2100397/storage/container_storage_interface/persistent-storage-csi.html#csi-drivers-supported_persistent-storage-csi

Additional information:

**PTAL**: @duanwei33 , @gcharot, @gnufied

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
LGTM.

**Comment by gnufied** [comment]
lgtm

**Comment by ahardin-rh** [comment]
LGTM

File: modules/persistent-storage-csi-drivers-supported.adoc [hunk 1]
```diff
@@ -28,8 +28,21 @@ The following table describes the CSI drivers that are installed with {product-t
 |OpenShift Data Foundation | ✅ | ✅ | ✅
 |OpenStack Manila | ✅ | - | -
 |Red Hat Virtualization (oVirt) | - | - | ✅
-|VMware vSphere | - | - | -
+|VMware vSphere | ✅^[1]^ | - | ✅^[2]^
 |===
+--
+1.
```
Comment (duanwei33) [inline modules/persistent-storage-csi-drivers-supported.adoc:34]: Not sure if this is more clear?
```suggestion
1. VMware vSphere CSI volume snapshots:
```

File: modules/persistent-storage-csi-drivers-supported.adoc [hunk 2]
```diff
@@ -28,8 +28,21 @@ The following table describes the CSI drivers that are installed with {product-t
...
+
+* Requires vSphere version 7.0 Update 3 or later for both vCenter Server and ESXi.
+
+* Does not support fileshare volumes.
+
+2.
```
Comment (duanwei33) [inline modules/persistent-storage-csi-drivers-supported.adoc:40]: And this one:
```suggestion
2. VMware vSphere CSI resize:
```

**Review by duanwei33 — COMMENTED** [review]
Looks good to me with minor suggestions.

File: modules/persistent-storage-csi-drivers-supported.adoc (hunk 1 above)
Comment (lpettyjo) [reply modules/persistent-storage-csi-drivers-supported.adoc:34]: See reply below

**Comment by gcharot** [comment]
LGTM

**Pull request merged**